packages/ui/src/layout/banners/Announcement.Countdown.tsx

```diff
@@ -10,19 +9,35 @@ interface CountdownStepProps {
   unit: string
 }
 
+interface CountdownButtonProps {
+  dest: string
+  text: string
+}
+
 const announcement = _announcement as AnnouncementProps
 
 function CountdownStep({ value, unit }: CountdownStepProps) {
   return (
-    <div className="rounded-md p-[1px] overflow-hidden bg-gradient-to-b from-[#FFFFFF50] to-[#FFFFFF00]">
-      <div className="py-1 px-2 rounded-md leading-4 flex items-center justify-center bg-gradient-to-b from-[#9E44EF40] to-[#DBB8BF40] backdrop-blur-md">
-        <span className="m-0">{value}</span>
+    <div className='rounded-md p-[1px] overflow-hidden bg-gradient-to-b from-[#FFFFFF50] to-[#FFFFFF00]'>
+      <div className='py-1 px-2 rounded-md leading-4 flex items-center justify-center bg-gradient-to-b from-[#9E44EF40] to-[#DBB8BF40] backdrop-blur-md'>
+        <span className='m-0'>{value}</span>
         <span>{unit}</span>
       </div>
     </div>
   )
 }
 
+function CountdownButton({ dest, text }: CountdownButtonProps) {
+  const classNames = 'bg-white text-xs px-1.5 md:px-2.5 py-1 rounded-full text-[#9E44EF] shadow-none hover:shadow-mg cursor-pointer';
+  return (
+    <div onClick={(e) => e.stopPropagation()}>
```

Comment:
Curious why we need the wrapping `div`?

Reply:
I think this was leftover from when I had the logic for using <Link> if its www and <a> if its in docs, removed in latest push :)